development-docs/DEV_GUIDE.md

```diff
@@ -382,21 +382,28 @@ the commit if errors are detected:
 ./tools/git-hooks/checkstyle-pre-commit
 ```
 
-## Building container images for other platforms with Docker `buildx`
+## Building container images for other platforms
 
-Docker supports building images for different platforms using the `docker buildx` command. If you want to use it to
-build Strimzi images, you can just set the environment variable `DOCKER_BUILDX` to `buildx`, set the environment
-variable `DOCKER_BUILD_ARGS` to pass additional build options such as the platform and run the build. For example
-following can be used to build Strimzi images for Linux on Arm64 / AArch64:
+Docker and Podman supports building images for different platforms. Strimzi relies on `TARGETOS` and `TARGETARCH` 
+arguments to identify which platform of binary should be used when building the image. Please make sure the version of 
+docker or podman used is supporting this feature. If you want to build Strimzi images for other platforms, you can set 
```

Comment:
Adding these lines to mention we relied on `TARGETOS` and `TARGETARCH` to build different platform. This issue in podman: https://github.com/containers/podman/issues/14375 showed, these 2 arguments support starts at 4.2.0 and later. I believe old docker won't support them, either.